apps/prairielearn/src/pages/assessmentsSwitcher/assessmentsSwitcher.html.ts

```
@@ -0,0 +1,158 @@
+                    </span>
+                  `
+                : ''}
+            </p>
+            <p class="m-0 ${isActive ? 'text-light' : 'text-muted'} small">
+              ${assessmentDropdownItemData.tid}
+            </p>
+          </div>
+        </a>
+      `;
+    })}
+  `.toString();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)